packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -197,11 +197,23 @@ export function getClientIpHeader(request) {
 	}
 }
 
+export function getTrackingHeaders(request) {
+	// email tracking with _xtd query param: https://meetup.atlassian.net/wiki/spaces/DAT/pages/27754630/Email+Tracking
+	var trackingParam = request.query._xtd;
+	if (trackingParam) {
+		return {
+			"X-Meetup-External-Track": trackingParam,
+			"X-Meetup-External-Track-Url": request.url.href,
+		};
+	}
```

Comment:
just to keep return types consistent, I'd recommend returning an empty object here - i.e. the function always returns an object rather than an `?Object`

Reply:
should that be added to 
```
export function getClientIpHeader(request) {
	const clientIP =
		request.query['_set_geoip'] || request.headers['fastly-client-ip'];
	if (clientIP) {
		return { 'X-Meetup-Client-Ip': clientIP };
	}
}
```

Reply:
sure!